src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/Instruction.java

```diff
@@ -20,7 +20,7 @@
 import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
 import me.mrCookieSlime.Slimefun.api.inventory.BlockMenu;
 
-enum Instruction {
+public enum Instruction {
```

Comment:
why?

Reply:
As I already said, so addons can modify android code

Reply:
This is an enum though...

Reply:
I plan to generate an Instruction[] in my addon code, then turn it into a string that the android recognises

Reply:
Okay, I thought you meant adding more Instructions, because for that we would need an interface (which we should add at some point anyway)